src/utilities/permissions.ts

```diff
@@ -1250,6 +1250,9 @@ const gatewayPermissions = {
       )
     );
   },
+  IMPORT_SEQUENCE_TEMPLATE: (user: User | null) => {
```

Comment:
Does this need an accompanying `featurePermission` to disallow the user from clicking on any buttons to import templates that they don't have permissions for?

Reply:
yep good catch, thanks 👍